build/stf-run-ci/tasks/create_catalog.yml

```diff
@@ -7,11 +7,57 @@
 
 # Updating to use stdout_lines[-1] so that any additional info that gets added to generate_bundles (e.g. for debug) doesn't break this task
 # Adding from_json so that the JSON output is parsed into a dictionary
-- name: Create info variables from bundle generation output
+- name: Create info variables from bundle generation output (local build)
+  when: __local_build_enabled | bool
   ansible.builtin.set_fact:
     sto_bundle_info: "{{ generate_bundle_sto.stdout_lines[-1] | from_json }}"
     sgo_bundle_info: "{{ generate_bundle_sgo.stdout_lines[-1] | from_json }}"
 
+- name: Create info variables from provided pre-built bundles (deploy from bundles)
+  when: __deploy_from_bundles_enabled | bool
+  block:
+    - name: Define channel for pre-built bundles
+      ansible.builtin.set_fact:
+        stf_channel: stable-1.5
+    - name: Set info variables from provided pre-built bundles
+      ansible.builtin.set_fact:
+        sto_bundle_info:
+          'bundle_default_channel': "{{ stf_channel }}"
+          'bundle_channels': "{{ stf_channel }}"
+          'operator_bundle_version': "{{ sto_bundle_image_tag }}"
+        sgo_bundle_info:
+          'bundle_default_channel': "{{ stf_channel }}"
+          'bundle_channels': "{{ stf_channel }}"
+          'operator_bundle_version': "{{ sgo_bundle_image_tag }}"
+
+- name: Show STO and SGO bundle info that will used in the index image
+  when: __local_build_enabled | bool or __deploy_from_bundles_enabled | bool
+  ansible.builtin.debug:
+    msg:
+    - "{{ sto_bundle_info }}"
+    - "{{ sgo_bundle_info }}"
```

Comment:
Is deploy_from index AND not deploy_from_bundle AND not local_build a valid option?

IF it is not, then this should not need a `when: __local_build_enabled | bool or __deploy_from_bundles_enabled | bool`

Reply:
Another good catch